lambdas/replay-cimit-vcs/dynamo-scripts/scan.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+Help()
+{
+   # Display Help
+   echo "Syntax: scriptTemplate [-e|p]"
+   echo "options:"
+   echo "e     Specifies the environment label for the script output"
+   echo "p     Specifies the AWS profile to use with the script"
+   echo
+}
+
+# Set variables
+Env="build"
+Profile="None"
+
+# Script options
+while getopts ":he:p:" option; do
+   case $option in
+      h) # display Help
+         Help
+         exit;;
+      e) # Enter an environment
+         Env=$OPTARG;;
+      p) # Enter an AWS profile
+         Profile=$OPTARG;;
+     \?) # Invalid option
+         echo "Error: Invalid option"
+         exit;;
+   esac
+done
+
+echo Performing scan for $Profile in $Env
+
+# Scan script
+aws dynamodb scan \
+--table-name user-issued-credentials-v2-$Env \
+--filter-expression "attribute_exists(dateCreated) AND credentialIssuer = :vc" \
+--projection-expression "#ci, #dc, #uid" \
+--expression-attribute-names file://expression-attribute-names.json \
+--expression-attribute-values file://expression-attribute-values-address.json > ./$Env/address-results.json \
+--profile $Profile &&
```

Comment:
Do we need to chain these together with `&&` if it's in a script? Can we just run them as sequential steps?

You might need to set the 'e' flag at the top (i.e. `set -e`) so it exits if any command fails

Reply:
If you're feeling fancy you could create a shared function so it's clearer that it's doing the same thing for each CRI. Coding in bash is a pain though :(